modules/gateway/peers.go

```diff
@@ -35,6 +33,19 @@ var (
 			panic("unrecognized build.Release")
 		}
 	}()
+	// the gateway will abort a connection attempt after this long
+	dialTimeout = func() time.Duration {
+		switch build.Release {
+		case "dev":
+			return 2 * time.Minute
```

Comment:
I can't remember why I added this commit. I think it was because a test was failing and I suspected it might have been due to a long `dialTimeout`. This change really belongs in another PR. I'll remove it from this PR since there's a bunch of constants to update and they can all go in the same PR.